rizzino_a10.js

```diff
@@ -0,0 +1,21 @@
+/*
+Julie Rizzino
+rizzino_a10.js
+INFO 2124
+Thoendel
+2/18/20 */
+
+document.getElementById('action').addEventListener('click', function(){
+    //WRITE YOUR SOLUTION BETWEEN THIS COMMENT
+    document = "index.html";
+    //Declare variable document as the index.html 
+    const img = document.getElementById("photo");
+    //Declare constant img to equal the ID "photo" using the getElementById feature
+    if(img.src != "02.jpg"){
```

Comment:
hrm...this is not how we covered this in the presentation....try img.getAttribute("src") instead...it will work better.